docs/flow/config-language/expressions/operators.md

```diff
@@ -6,3 +6,97 @@ weight: 300
 ---
 
 # Operators
+River uses a set of operators that most should be familiar with. All operations
+follow the standard [PEMDAS](https://en.wikipedia.org/wiki/Order_of_operations)
+rule for operator precedence.
+
+## Arithmetic operators
+```
++    for addition: 3+5
+-    for subtraction: 10-3
+*    for multiplication: 13*7
+/    for division: 15/3
+%    for remainder: 7%2
+^    for exponentiation: 2^8
+```
+
+The `+` operator can also be used for string concatenation.
+
+## Comparison operators
+```
+==    equal
+!=    not equal
+<     less
+<=    less or equal
+>     greater
+>=    greater or equal
+```
+
+The equality operators `==` and `!=` can be applied to any operands.
+
+On the other hand, for the ordering operators the two operands must both be
+_orderable_ and of the same type. These terms and the result of the comparisons
+are defined as follows:
```

Comment:
You use the term "ordering operators" but don't define it here. Can we explain which operators are ordering operators?